html/about.html

```diff
@@ -8,60 +8,130 @@
 </head>
 
-    <body>
-        {{template "navbar"}}
+<body>
+    {{template "navbar"}}
 
-        <div class="container">
-            {{if ne .ErrorMessage ""}}
-            <div class="alert alert-danger">
-                {{.ErrorMessage}}
-            </div>
-            {{end}}
+    <div class="container">
+        {{if ne .ErrorMessage ""}}
+        <div class="alert alert-danger">
+            {{.ErrorMessage}}
+        </div>
+        {{end}}
+
+        <div class="container pt-5 pl-0">
+            <h2>About<br></h2>
+        </div>
+        <hr>
 
-            <div class="container pt-5 pl-0">
-                <h2>Installed Packages<br></h2>
-            </div>
 
-            <div class="row">
-                <div class="col-xs-12" >
-                    <div class="table-responsive">
-                        <table class="table">
-                            <thead>
-                                <tr>
-                                    <th>Package</th> 
-                                    <th>Version</th> 
-                                    <th></th>
-                                    <th></th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                                {{range .PackageDataRows}}
-                                    <tr>
-                                        <td>{{index . 0}}</td>       
-                                        <td class="text-center">{{index . 1}}</td>   
-                                        <td></td>    
-                                        <td></td>    
-                                    </tr>
-                                {{end}}
-                            </tbody>
-                        </table>
-                    </div>
+        <div class="container pt-5 pl-0">
+            <h3>Device<br></h3>
+        </div>
+
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="table-responsive">
+                    <table class="table">
+                        <tbody>
+                            <tr>
+                                <td>Name</td>
+                                <td>{{DeviceName}}</td>
+                                <td></td><td></td>
+                            </tr>
+                            <tr>
+                                <td >Group</td>
+                                <td id="Group"></td>
+                                <td></td><td></td>
+                            </tr>
+                            <tr>
+                                <td>Serial Number</td>
+                                <td>{{.RaspberryPiSerialNumber}}</td>
+                                <td></td><td></td>
+                            </tr>
+                            <tr>
+                                <td>API Identifier</td>
+                                <td id="APIID"></td>
+                                <td></td><td></td>
```

Comment:
Why are there these blank columns at the end of each row?

Reply:
Well it's because this is the best way that I have found to get the 2 columns with info in them to size and resize on all devices.

In Bootstrap you can't really specify the width of a column in a table.  So this makes things awkward.

This seems to be the easiest solution. I've used it on other pages here too.

Reply:
You should be able to wrap the whole table in div with a bootstrap "col-*" class to control the width of the table. This will then constrain the columns within the table. 

Failing that there's 2 more approaches here that all seem reasonable: https://stackoverflow.com/questions/31318467/table-cell-width-with-bootstrap